clifford/_layout.py

```diff
@@ -447,7 +447,7 @@ def dual_func(self):
         else:
             # Equivalent to but faster than
             #   Iinv = self.pseudoScalar.inv().value
-            Iinv = np.zeros(self.gaDims)
+            Iinv = np.zeros(self.gaDims, dtype=int)
```

Comment:
@eric-wieser do you think this is a safe thing to do? I was aiming for a type that could be upgraded to whatever the type of the multivector was

Reply:
Ideally you'd pick the type based on the type of `1 / II_scalar`

Reply:
Otherwise this will go wrong for metrics containing 2